.circleci/config.yml

```diff
@@ -464,42 +650,56 @@ workflows:
             branches:
               only: master
 
+  # release-branch workflow is triggered when branch is "release-*"
+  # and tag is not empty
```

Comment:
hmm, this is when we create or update release branches? In those case, we don't use tags. I think the comment is not accurate, right? I mean, I think the condition in the `when` is right,  but this comment says the opposite, right?